test/cpp/qps/server_async.cc

```diff
@@ -336,7 +332,7 @@ class AsyncQpsServerTest : public Server {
   std::unique_ptr<grpc::Server> server_;
   std::vector<std::unique_ptr<grpc::ServerCompletionQueue>> srv_cqs_;
   ServiceType async_service_;
-  std::forward_list<ServerRpcContext *> contexts_;
+  std::vector<std::unique_ptr<ServerRpcContext>> contexts_;
```

Comment:
Yes, `>>` is ok for ending nested template declarations in all the gcc, clang, and VS versions that we support.